src/sharect.js

```diff
@@ -116,16 +138,28 @@ const Sharect = (function(){
 
       div.appendChild(arrow)
 
-      document.body.appendChild(div)
+      if (hasSelection()) {
+        offsetParent(_selection.anchorNode.parentElement).appendChild(div)
```

Comment:
This is the only line needed here.
`drawTooltip` is called after a `hasSelection` test in the `mouseup` event, so no need to test again here.
`offsetParent` already returns the `offsetParent || document.body`